install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -0,0 +1,188 @@
+prometheus-k8s-0                               4/4       Running             1          35m
+prometheus-k8s-1                               0/4       ContainerCreating   0          21s
+prometheus-operator-6c9fddd47f-9jfgk           1/1       Running             0          36m
+----
+
+. Open the configuration file for the cluster monitoring stack:
++
+[subs="quotes"]
+----
+$ *oc -n openshift-monitoring edit configmap cluster-monitoring-config*
+----
+
+. Under `config.yaml: |+`, add the `etcd` section.
++
+If you use static pods, you can specify the `etcd` nodes using the selector:
++
+[subs="quotes"]
+----
+...
+data:
+  config.yaml: |+
+    ...
+    *etcd:
+      targets:
+        selector:
+          openshift.io/component: etcd
+          openshift.io/control-plane: "true"*
+----
++
+If you do not use static pods, you need to specify the nodes using IP addresses:
++
+[subs="quotes"]
+----
+...
+data:
+  config.yaml: |+
+    ...
+    *etcd:
+      targets:
+       ips:
+       - "127.0.0.1"
+       - "127.0.0.2"
+       - "127.0.0.3"*
+----
++
+If `etcd` nodes IP addresses change, you need to update this list.
+
+. Verify that the `etcd` service monitor is now running:
++
+[subs="quotes"]
+----
+$ *oc -n openshift-monitoring get servicemonitor*
+NAME                  AGE
+alertmanager          35m
+*etcd                  1m*
+kube-apiserver        36m
+kube-controllers      36m
+kube-state-metrics    34m
+kubelet               36m
+node-exporter         34m
+prometheus            36m
+prometheus-operator   37m
+----
++
+It might take up to a minute for the `etcd` service monitor to start.
+
+. Now you can navigate to the Web interface to see more information about status of `etcd` monitoring. To get the URL, run:
++
+[subs="quotes"]
+----
+$ *oc -n openshift-monitoring get routes*
+NAME                HOST/PORT                                                                           PATH      SERVICES            PORT      TERMINATION   WILDCARD
+...
+prometheus-k8s      prometheus-k8s-openshift-monitoring.apps.msvistun.origin-gce.dev.openshift.com                prometheus-k8s      web       reencrypt     None
+----
++
+Using `https`, navigate to the URL listed for `prometheus-k8s`. Log in.
+
+. Ensure the user belongs to the `cluster-monitoring-view` role. This role provides access to viewing cluster monitoring UIs. For example, to add user `developer` to `cluster-monitoring-view`, run:
+
+  $ oc adm policy add-cluster-role-to-user cluster-monitoring-view developer
++
+
+. In the Web interface, log in as the user belonging to the `cluster-monitoring-view` role.
+
+. Click `Status`, then `Targets`.
+
+. You should see the `etcd` entry. This means that `etcd` is being monitored.
++
+image::etcd-no-certificate.png[]
+
+However, Prometheus is not yet able to authenticate against `etcd`, and so cannot gather metrics. Follow these steps to configure Prometheus authentication against `etcd`:
```

Comment:
If this is the general process to add Prometheus auth, I'd make it a separate section under a separate heading. If not, consider making the following steps substeps of this one. I'd replace 119 with "To configure Prometheus authentication against `etcd` so it can gather metrics:"